rgdps/usecases/messages.py

```diff
@@ -0,0 +1,269 @@
+from __future__ import annotations
+
+from datetime import datetime
+from typing import NamedTuple
+
+from rgdps import repositories
+from rgdps.common.context import Context
+from rgdps.constants.errors import ServiceError
+from rgdps.constants.users import UserPrivacySetting
+from rgdps.constants.users import UserRelationshipType
+from rgdps.models.message import Message
+from rgdps.models.user import User
+
+
+class MessageResponse(NamedTuple):
+    message: Message
+    user: User
+
+
+class PaginatedMessagesResponse(NamedTuple):
+    messages: list[MessageResponse]
+    total: int
+
+
+async def from_id(
```

Comment:
I would probably call this something along the lines of `get` for consistency.

Reply:
resolved in 8109c21